TestTime fails

TestTime test fails with go 1.14.2 in UTC+1 timezone.

```
$ go version
go version go1.14.2 linux/amd64
$ go test -v
=== RUN   TestTime
    TestTime: php_test.go:312: Expected 27/04/2018 11:23:14 AM (type string) - Got 27/04/2018 04:23:14 AM (type string)
--- FAIL: TestTime (0.00s)
$ 
```
The reason is likely that time.Unix() uses local time input, which means that test will faily every time it is run outside of author's timezone.

Test:
```
equal(t, "27/04/2018 11:23:14 AM", Date("02/01/2006 15:04:05 PM", 1524799394))
```

Code: 
```
func Date(format string, timestamp int64) string {
	return time.Unix(timestamp, 0).Format(format)
}
```

This should ideally be fixed so tests can be run successfully in any timezone.

--- a/php_test.go
+++ b/php_test.go
@@ -7,13 +7,17 @@ import (
 	"os"
 	"reflect"
 	"testing"
+	"time"
 	"unicode/utf8"
 )
 
 func TestTime(t *testing.T) {
 	gt(t, float64(Time()), 1522684800)
 
-	equal(t, "27/04/2018 11:23:14 AM", Date("02/01/2006 15:04:05 PM", 1524799394))
+	// Ensure we take timezones into account
+	_, offset := time.Now().Local().Zone()
+	unix := int64(1524799394)
+	equal(t, "27/04/2018 03:23:14 AM", Date("02/01/2006 15:04:05 PM", unix-int64(offset)))
 
 	tStrtotime, _ := Strtotime("02/01/2006 15:04:05", "02/01/2016 15:04:05")
 	equal(t, int64(1451747045), tStrtotime)

